tests/integration/scenarios/idle.js

```diff
@@ -199,12 +199,15 @@ describe("initial idle state", () => {
         player.setVolume(1);
 
         expect(player.mute()).to.equal(undefined);
-        expect(player.getVolume()).to.equal(0);
-        expect(videoElement.volume).to.equal(0);
-        expect(videoElement.muted).to.equal(false);
+        expect(player.getVolume()).to.equal(1);
+        expect(videoElement.volume).to.equal(1);
+        expect(videoElement.muted).to.equal(true);
         expect(player.isMute()).to.equal(true);
         player.unMute();
         expect(player.isMute()).to.equal(false);
```

Comment:
is this test still usefull with the changes you made ? 
At least we should change the test descriptions as "should set the volume to 0 in mute by default" no longer describes what's currently done in that test

Reply:
Yes you're right I forgot to update the test title.

For the rest, it could technically still make sense to check that the volume is not affected by muting

Reply:
I renamed the test

Reply:
Nice